docs/installation.md

```diff
@@ -15,6 +16,9 @@ First, add our official ralph repository:
     sudo apt-key adv --keyserver  hkp://keyserver.ubuntu.com:80 --recv-keys 379CE192D401AB61
     sudo sh -c "echo 'deb https://dl.bintray.com/vi4m/ralph wheezy main' >  /etc/apt/sources.list.d/vi4m_ralph.list"
 
+After, install the secure transport for accessing repository :
```

Comment:
I suppose it can be placed before adding the cert key. 
But on my side I downloaded the key before and that created no issue.
The only trouble occurs if you run apt-get update before installing the secure transport.